<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Frame" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>681</height>
    </rect>
   </property>
   <widget class="QWidget" name="RgbWidget" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QLabel" name="RgbLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="rgbCopyWidget" native="true">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QPushButton" name="rgbCopy">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPusButton:pressed{
background-color : rgb(81, 161, 0);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>copyIcon.png</normaloff>copyIcon.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>RGB:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="BgrWidget" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QLabel" name="BgrLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="bgrCopyWidget" native="true">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QPushButton" name="bgrCopy">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>copyIcon.png</normaloff>copyIcon.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>BGR:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="HexWidget" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>220</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QLabel" name="HexLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="hexCopyWidget" native="true">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QPushButton" name="hexCopy">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>copyIcon.png</normaloff>copyIcon.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HEX:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="generateFrame" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QWidget" name="generateButtonWidget" native="true">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QPushButton" name="generateButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Constantia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="generateLineWidget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="generateLine">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="colorComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>RGB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BGR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HEX</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HSV</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="PictureWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QLabel" name="PictureLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>611</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>             Please Select a Picture to  Analyze</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="fileBrowserWidget" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>530</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <widget class="QPushButton" name="fileBrowserButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>fileBrowserIcon.png</normaloff>fileBrowserIcon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="colorPaletteWidget" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>550</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="HsvWidget" native="true">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QLabel" name="HsvLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="hsvCopyWidget" native="true">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QPushButton" name="hsvCopy">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>copyIcon.png</normaloff>copyIcon.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HSV:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mouseCurrentWidget" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>490</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>580</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>   ↑</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>600</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse your own image</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
